fishproviz/trajectory/feeding_shape.py

```diff
@@ -89,36 +89,44 @@ def get_feeding_cords(data, camera_id, is_back):
 
 
 def get_feeding_box(data, TL_x, TL_y, TR_x, TR_y):
-    scale = 1
+    cm_len = config.MAGNET_LENGTH_CM
     # if x has the same value.
     if abs(TL_x - TR_x) < abs(TL_y - TR_y):
         # config.FRONT
-        p_len = abs(TL_y - TR_y) * scale
-        f1 = data["xpx"] > TL_x - (2 * p_len)
-        f2 = data["ypx"] < TL_y + (p_len/2)
-        f3 = data["ypx"] > TR_y - (p_len/2)
+        p_len = abs(TL_y - TR_y)
+        target_width = config.FEEDING_SHAPE_WIDTH * p_len / cm_len
+        target_height = config.FEEDING_SHAPE_HEIGHT * p_len / cm_len
```

Comment:
clarification needed: normalizing the respective side with `p_len/cm_len` might be 
a) unneccessary and convoluted, as the size of the magnet (shouldn't) deviate from 2.5cm, and we are here normalizing with the factor 1 in any case if we always account for the correct size and specify it in the config; and
b) misleading, as a different size of the magnet (e.g. p_len = 5cm) wouldn't normalize the width to 10cm and the height to 13cm - if specified so, but would lead to width = 20cm and height = 26cm, if we would not account for the correct size in the config (e.g. leave cm_len = 2.5)
---
please elaborate on what the heuristic here might be; otherwise, let's omit the `MAGNET_LENGTH_CM` env-var.

Reply:
The p_len extracted from `data/feeding_patch_coords.csv` is in pixels. Take for example this entry:

```
camera_id;front_or_back;TL_x;TL_y;TR_x;TR_y
23442333;front;3510;2264;3510;2149
```

`p_len = abs(TL_y - TR_y)` then yields 115 pixels. To be able to tell how many pixels correspond to 1cm and therefore convert the user-provided target shape from metric to pixels, we would need the magnet length, `cm_len` which ultimately corresponds to the conversion of `p_len` to the metric system.